core/src/main/scala/org/apache/gearpump/cluster/worker/Worker.scala

```diff
@@ -199,7 +199,10 @@ private[cluster] object Worker {
         val classPathPrefix = Util.getCurrentClassPath ++ ctx.classPath
         val classPath = jarPath.map(classPathPrefix :+ _).getOrElse(classPathPrefix)
 
-        val logArgs = List(s"-D${Constants.GEARPUMP_APPLICATION_ID}=${launch.appId}", s"-D${Constants.GEARPUMP_EXECUTOR_ID}=${launch.executorId}")
+        val logArgs = List(
+          s"-D${Constants.GEARPUMP_APPLICATION_ID}=${launch.appId}",
+          s"-D${Constants.GEARPUMP_EXECUTOR_ID}=${launch.executorId}",
+          s"-D${Constants.GEARPUMP_APPLICATION_TIMESTAMP}=${Util.getCurrentTime()}")
```

Comment:
Executor of same app on different worker will have different timestamp?

We need a global consistent view of timestamp, better this is the time when the app is submitted to master.